classes/rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :book, :person, :date
+
+  def initialize(date, person = nil, book = nil)
+    @person = person
+    @book = book
+    @date = date
+  end
```

Comment:
* Kindly add the association `a rental belongs to a book` by adding itself(rental) to the @book achieving the 'belongs-to' association also the same thing for this association `a rental belongs to a person` as [required](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_associations.md#:~:text=Create%20the%20has%2Dmany%20side%20of,a%20rental%20belongs%20to%20a%20person). Hint below 👇 
 ```ruby
@book = book
    book.rentals << self

    @person = person
   
    person.rentals << self
```

Reply:
Implemented